x-pack/plugins/infra/common/inventory_models/container/metrics/tsvb/container_k8s_overview.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TSVBMetricModelCreator, TSVBMetricModel } from '../../../types';
+
+export const containerK8sOverview: TSVBMetricModelCreator = (
+  timeField,
+  indexPattern,
+  interval
+): TSVBMetricModel => ({
+  id: 'containerK8sOverview',
+  requires: ['kubernetes.container'],
+  index_pattern: indexPattern,
+  interval,
+  time_field: timeField,
+  type: 'timeseries',
+  series: [
+    {
+      id: 'cpu',
+      split_mode: 'everything',
+      metrics: [
+        {
+          field: 'kubernetes.container.cpu.usage.node.pct',
+          id: 'avg-cpu-total',
+          type: 'avg',
+        },
+      ],
+    },
+    {
+      id: 'memory',
+      split_mode: 'everything',
+      metrics: [
+        {
+          field: 'kubernetes.container.memory.usage.node.pct',
```

Comment:
I think it would be better to use `kubernetes.container.memory.usage.limit.pct` here instead.
It will show the percentage of the limit that was assigned to the container instead or the node if a limit hasn't been set.